packages/api/src/SubmittableExtrinsic.ts

```diff
@@ -3,8 +3,9 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { KeyringPair } from '@polkadot/keyring/types';
-import { AccountId, Address, Extrinsic, ExtrinsicStatus, EventRecord, Hash, Index, Method, SignedBlock, Struct, Text, Vector } from '@polkadot/types/index';
-import { Codec, CodecCallback, SignatureOptions } from '@polkadot/types/types';
+import { AccountId, Address, ExtrinsicStatus, EventRecord, Hash, Index, Method, SignedBlock, Struct, Text, Vector } from '@polkadot/types/index';
+import { Codec, CodecCallback, ExtrinsicLike, SignatureOptions } from '@polkadot/types/types';
+import getRegistry from '@polkadot/types/codec/typeRegistry';
```

Comment:
Just export `getRegistry` in `@polkadot/codec/index` (so it is available everywhere as part of `@polkadot/types`)